jellyfin_auto_skip.py

```diff
@@ -179,4 +179,4 @@ def receiveSignal(signalNumber, frame):
 
 if __name__ == "__main__":
     signal.signal(signal.SIGINT, receiveSignal)
-    main(sys.argv[1:])
```

Comment:
I'm not sure what's going on with this line. Can you fix it so only the affected line (g_monitor_all_users) is changed?

Reply:
Idk, I didn't do anything on that line. Can you manually change it and close the request?

Reply:
I just tried changing it again in a new branch and it still automatically does it